File: languages/go/entity/identity/holder.go
package identity

import (
	"github.com/bloock/bloock-sdk-go/v2/entity/key"
)

// Holder represents a Holder identity.
type Holder struct {
	Did Did
	Key key.Key
}

// NewHolder returns a new instance of Holder identity for the given parameters.
func NewHolder(did string, didMethod DidMethod, key key.Key) Holder {
	return Holder{
		Did: NewDid(did, didMethod),
		Key: key,
	}
}

File: languages/js/src/entity/identity/holder.ts
import { Key } from "../key/key";
import { Did } from "./did";
import { DidMethod } from "./did_method";

/**
 * represents a Holder identity.
 */
export class Holder {
    did: Did;
    key: Key;

    /**
     * Constructs a Holder object with the specified parameters.
     * @param did 
     * @param didMethod 
     */
    constructor(did: string, key: Key, didMethod: DidMethod) {
        this.did = new Did(did, didMethod);
        this.key = key;
    }
}